src/els_code_navigation.erl

```diff
@@ -51,9 +51,8 @@ goto_definition(Uri, #{ kind := record_expr, id := Record }) ->
 goto_definition(_Uri, #{ kind := Kind, id := Include }
                ) when Kind =:= include;
                       Kind =:= include_lib ->
-  FileName = filename:basename(Include),
-  M = list_to_atom(FileName),
-  case els_utils:find_module(M) of
+  H = list_to_atom(filename:basename(Include, filename:extension(Include))),
+  case els_utils:find_header(H) of
```

Comment:
This is something that annoyed me for a long time. We were looking for modules named 'xxx.hrl'.